cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -19,31 +19,28 @@ package app
 import (
 	"context"
 	"fmt"
-	"net"
+	"net/http"
 	"os"
-	"strings"
 	"time"
 
 	"github.com/spf13/cobra"
 
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apiserver/pkg/server"
 	"k8s.io/apiserver/pkg/server/healthz"
 	apiserverflag "k8s.io/apiserver/pkg/util/flag"
 	"k8s.io/apiserver/pkg/util/globalflag"
-	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/leaderelection"
 	"k8s.io/client-go/tools/leaderelection/resourcelock"
-	cloudprovider "k8s.io/cloud-provider"
```

Comment:
I'd keep this so one knows what it's imported as.

Reply:
you are right, fixed